pages/inspector/inspector-debugger.mdx

```diff
@@ -0,0 +1,100 @@
+import { Callout } from 'nextra/components'
+
+# Inspector Debugger
+
+The Inspector Debugger is a real-time event stream viewer that lets you validate your tracking implementation as you develop, without waiting for events to flow through your analytics pipeline. See events as they happen, inspect their properties, and catch issues before they reach production.
```

Comment:
Perhaps also highlight QA and documentation (what's the event for...) use cases here?